docs/platform/collaboration.md

```diff
@@ -12,14 +12,41 @@ Available features include and not limited to:
 | Activity | Yes/No |
 | :--- | :--- |
 | Using your own cloud credentials | Yes |
-| Download team member’s artifacts | Yes |
+| View the team information | Yes |
+| Download team member’s artifacts  | Yes |
 | Visualize metrics of team member’s Run | Yes |
-| Clone or share a team member’s Run/Experiment | Yes |
 | View logs of team member’s Runs | Yes |
 | View Tensorboard outputs of team member's Run | Yes |
 | View team member's Sessions | Yes |
+| Pause a session created by another team member | No |
+| Delete a run created by another team member | No |
+| Access session created by team member using ssh  | No |
+| Access session created by team member using vscode | No |
 | View team member's Datastores | Yes |
+| Stop a run created by another team member | Yes |
+| Delete a run created by another team member | Yes |
 | Access JupyterLab session created by a team member | Yes |
 | Run with a Datastore created by you or any team member | Yes |
 | Mount a Datastore created by team member when creating session | Yes |
+| Add Credit for the team Payments | No |
 
+## Team Collaboration CLI commands
+Use the following command for checking out runs and experiments in the team
```

Comment:
@sunitaprakash 

1. instead of repeating the same line over and over ("use the following command for..") just simplify it to the action being taken

2. please use code syntax `like this` when including code snippets

Examples: 
Check status of Runs across team
`grid status --global`

View sessions on your team
`grid session --global`

Reply:
Hi! @essiequoi  I have made the changes .. thanks or the review